CommonTools/TriggerUtils/src/GenericTriggerEventFlag.cc

```diff
@@ -21,9 +21,19 @@ static const bool useL1EventSetup( true );
 static const bool useL1GtTriggerMenuLite( false );
 
 
+GenericTriggerEventFlag::GenericTriggerEventFlag( const edm::ParameterSet & config, edm::ConsumesCollector & iC):
+  GenericTriggerEventFlag(config,iC,false)
+{
+  if ( config.exists( "andOrL1" ) ) {
```

Comment:
Hi @Sam-Harper - given the GenericTriggerEventFlag(config,iC,false) call, is this needed?